persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAO.kt

```diff
@@ -14,7 +14,7 @@ data class ConversationEntity(
     val lastModifiedDate: String,
 ) {
 
-    enum class Type { SELF, ONE_ON_ONE, GROUP }
+    enum class Type { SELF, ONE_ON_ONE, GROUP, CONNECTION }
```

Comment:
Does this `CONNECTION` value also encapsulate cancelled connections? if not, maybe it would be more sensible to rename it to something like `CONNECTION_PENDING` or `INCOMING_CONNECTION`. Otherwise it may sound confusing as the other type of conversations are implied to have a connection